performance-test/results/results.md

```diff
@@ -0,0 +1,291 @@
+
+  sink:
+    - opensearch:
+        hosts: [ "myOpenSearchEndpoint" ]
+        index: "data_prepper"
+        username: "username"
+        password: "password"
+```
+
+## Logstash Configuration
+
+```
+input {
+    http {
+        ssl => false
+        port => 2021
+        codec => "json"
+	threads => 4
+    }
+}
+
+filter {
+    grok {
+        match => {"log" => "%{COMMONAPACHELOG}"}
+    }
+
+    mutate {
+        gsub => ["log", "\.", "-"]
+    }
+
+    mutate {
+        uppercase => [ "log" ]
+    }
+
+    mutate {
+        strip => [ "log" ]
+    }
+
+    mutate {
+        split => { "request" => "?" }
+    }
+
+    kv {
+        field_split => "&"
+        value_split => "="
+        source => "[request][1]"
+        target => "query_params"
+    }
+
+    mutate {
+        lowercase => [ "verb" ]
+    }
+
+    mutate {
+        add_field => {
+            "entry1" => "entry1value"
+            "entry2" => "entry2value"
+            "entry3" => "entry3value"
+        }
+    }
+
+    mutate {
+        rename => {
+            "entry1" => "renameEntry1"
+            "entry2" => "renameEntry2"
+            "entry3" => "renameEntry3"
+        }
+    }
+
+    mutate {
+        copy => {
+            "log" => "copy_key"
+        }
+    }
+
+    mutate {
+        remove_field => ["renameEntry1", "renameEntry2", "renameEntry3"]
+    }
+}
+
+output {
+    opensearch {
+        hosts => ["myOpenSearchEndpoint"]
+        user        => "user"
+        password    => "password"
+        ssl => true
+        ssl_certificate_verification => false
+        index       => "logstash"
+    }
+} 
+```
+
+The following changes were made to the `logstash.yml` file in order to replicate the Data Prepper configuration as much as possible
+
+```yaml
+pipeline.workers: 12
+pipeline.batch.size: 5000
+```
+
+## Results
+
+### Data Prepper
+
+Gatling reported the following results for the test that was run for Data Prepper. These results show that `118,104` requests were succesfully sent to Data Prepper. Throughput can be measured as 
+`118,104 * 200 (batch size) logs = 23,620,800 logs / 1,200 s = 19,684 logs/s`. The Gatling report also shows a distribution of latency on the side of the `http` source, with an average latency of `101 ms`
+
+![](../../docs/images/DataPrepperGatling.png)
+
+
+### Logstash
+
+Gatling reported the following results for the test that was run for Data Prepper. These results show that `75,709` requests were succesfully sent to Data Prepper. Throughput can be measured as
+`75,709 * 200 (batch size) logs = 15,141,800 logs / 1,200 s = 12,618 logs/s`. The Gatling report also shows a distribution of latency on the side of the `http` source, with an average latency of `158 ms`
+
+![](../../docs/images/LogstashGatling.png)
+
+When comparing these measurements side by side, the test shows that with this configuration, Data Prepper has a `56%` higher throughput than Logstash, and an average latency that is `56%` lower than Logstash.
```

Comment:
This like uses Data Prepper as the subject and should be in the section above. Or we can add a separate conclusion section as well. 

```
### Conclusion
Data Prepper has a `56%` higher throughput than Logstash, and an average latency that is `56%` lower than Logstash.
```